src/interface/src/app/shared/nav-bar/nav-bar.component.html

```diff
@@ -89,7 +89,11 @@
       class="action-button"
       (click)="sharePlan()"
       data-id="share"
-      *ngIf="area === 'SCENARIOS' && hasSharePlanFeatureFLag">
+      *ngIf="
+        area === 'SCENARIOS' &&
+        hasSharePlanFeatureFLag &&
+        (canShowSharePlan$ | async)
```

Comment:
nit: the name was confusing to me (and maybe it's only confusing to me :D ), but if this is just for the button, maybe we could call it `displaySharePlanBtn` or `displaySharePlanButton` or, alternatively, maybe just `canSharePlan` if we'd ever use this var for other things?

Reply:
yeah good point, not a great name. Will update, thanks!